awsappconfig-extension-demo/template.yml

```diff
@@ -41,8 +41,10 @@ Resources:
               - !Sub 'arn:aws:appconfig:${AWS::Region}:${AWS::AccountId}:application/${AppConfigApplication}'
```

Comment:
You can remove these other ARNs as the new data plane APIs just authorize on a single ARN